specs/commandline/inspect.md

```diff
@@ -0,0 +1,298 @@
+        ├── signed attributes
+        │   ├── content type: application/vnd.cncf.notary.payload.v1+json
+        │   ├── signing scheme: notary.signingAuthority.x509
+        │   ├── signing time: Fri Jun 23 22:04:01 2023
+        │   ├── expiry: Sat Jun 29 22:04:01 2024
+        │   └── io.cncf.notary.verificationPlugin: com.example.nv2plugin     
+        ├── unsigned attributes
+        │   ├── io.cncf.notary.timestampSignature: <Base64(TimeStampToken)> 
+        │   └── io.cncf.notary.signingAgent: notation/1.0.0   
+        ├── certificates
+        │   ├── SHA1 fingerprint: 68C15B4C98AD91E051EE5AF5F524A8729040B1D
+        │   │   ├── issued to: wabbit-com Software
+        │   │   ├── issued by: wabbit-com Software Root Certificate Authority
+        │   │   └── expiry: Sun Jul 06 20:50:17 2025
+        │   ├── SHA1 fingerprint: 4ACC2147712B3C555B1C96CFCC00215403TE011C
+        │   │   ├── issued to: wabbit-com Software Code Signing PCA
+        │   │   ├── issued by: wabbit-com Software Root Certificate Authority
+        │   │   └── expiry: Sun Jul 06 20:50:17 2025
+        │   └── SHA1 fingerprint: A4YA1205512B3C886B1C96AAEC89984914DC012A
+        │       ├── issued to: wabbit-com Software Root Certificate Authority
+        │       ├── issued by: wabbit-com Software Root Certificate Authority
+        │       └── expiry: Sat Jun 23 22:04:01 2035
+        └── signed artifact 
+            ├── media type: application/vnd.oci.image.manifest.v1+json
+            ├── digest: sha256:ca5427b5567d3e06a72e52d7da7dabfac484efe37a5380ee9088f7ace2eaab9
+            └── size: 16724
+```
+## Inspect signatures on the supplied OCI artifact with an example of JSON Output
+
+```shell
+notation inspect localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52da7dabfac484efe37a5380ee9088f7ace2efcde9 -o json
+```
+
+An example output:
+```jsonc
+{
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "signatures": [
+    {
+      "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a33",
+      "signatureAlgorithm": "RSASSA-PSS-SHA-256",
+      "signedAttributes": {
+        "contentType": "application/vnd.cncf.notary.payload.v1+json",
+        "signingScheme": "notary.default.x509",
+        "signingTime": "2022-02-06T20:50:17Z",
+        "expiry": "2023-02-06T20:50:17Z",
+        "io.cncf.notary.verificationPlugin": "com.example.nv2plugin"
+      },
+      "userDefinedAttributes": {
+        "io.wabbit-networks.buildId": "123"
+      },
+      "unsignedAttributes": {
+        "io.cncf.notary.timestampSignature": "<Base64(TimeStampToken)>",
+        "io.cncf.notary.signingAgent": "notation/1.0.0"
+      },
+      "certificates": [
+        {
+          "SHA1Fingerprint": "E8C15B4C98AD91E051EE5AF5F524A8729050B2A",
+          "issuedTo": "wabbit-com Software",
+          "issuedBy": "wabbit-com Software Root Certificate Authority",
+          "expiry": "Sun Jul 06 20:50:17 2025"
```

Comment:
The expiry datetime in JSON should be in the RFC3339 format
```suggestion
          "expiry": "2025-07-06T20:50:17Z"
```
Same comment applies to other lines of this JSON object.